Bang/Players/Player.cs

```diff
@@ -137,13 +143,15 @@ public void RegainLifePoint()
             PlayerTablet.Health++;
         }
 
-        public void LoseLifePoint()
+        public void LoseLifePoint(int loseLifeAmount = 1)
         {
-            Debug.Assert(PlayerTablet.Health > 0);
-            
-            PlayerTablet.Health--;
-        }
+            for (int i = 0; i < loseLifeAmount; i++)
+            {
+                Debug.Assert(PlayerTablet.Health > 0);
```

Comment:
Scenario: dynamite explodes on player with one health point. The debug assertion will be failed in the second iteration.

So I suppose to put debug assertion into method start.

Reply:
done